recipe/meta.yaml

```diff
@@ -38,14 +39,18 @@ test:
     - python-build --help 
   requires:
     - pip
-    - python <3.10
+    - python
```

Comment:
`    - python` can be removed from `test/requires `because it's already in `requirements/run`

Reply:
Removed!